Sources/PointFree/Routers/PointFreeRoutes.swift

```diff
@@ -76,9 +78,19 @@ private let routers: [Router<Route>] = [
   Route.iso.confirmCancel
     <¢> get %> lit("account") <% lit("cancel"),
 
+  Route.iso.confirmEmailChange
+    <¢> get %> lit("account") %> lit("confirm-email-change")
+    %> queryParam("payload", .appDecrypted >>> payload(.uuid >>> .tagged, .tagged))
```

Comment:
ok, lots of new combinators here! :D 

so first, theres the `appDecrypted: PartialIso<String, String>` iso that simply decrypts in the `apply` direction and `encrypts` in the `unapply` direction. it uses the `appSecret` for encryption under the hood, hence why I called it `appDecrypted`. you can compose this guy with any others to automatically encrypt a param. for example, you could do `.appDecrypted >>> .uuid >>> .tagged` to have an encrypted, tagged UUID. pretty amazing!

then there's this `payload` combinator. it has the shape `(PartialIso<String, A>, PartialIso<String, B>) -> PartialIso<String, (A, B)>`. it's job us to combine the results of two isos into a single string with a separator. For example, it allows for splitting a string like `123--true` into an `Int` and a `Bool`. the idea is it can combine multiple values into a single string payload.

the reason we need this is that we cannot simply have an encrypted user id and encrypted email floating around separately in the url. a user could then mix and match encrypted values from multiple emails to allow changing emails we did not mean to allow. by packing both values into a single payload and then encrypting that we prevent this attack.

Reply:
so cool!